### ISSUE TITLE
SSH | Syntax error: Bad function name

### ISSUE THREAD

**Issue by Fourdee** [MichaIng/DietPi#1776]
> https://github.com/Fourdee/DietPi/pull/1771#issuecomment-388696242

Notes:
- Unable to replicate with putty/kitty.
- Unable to replicate with ```openssh-client``` on a DietPi system.

**Comment by Fourdee** [comment]
@slim0287

Question:
Can you list all the system changes you've made that you can remember (eg: locale set to en_US, timezone changed, installed nextcloud)?

**Comment by slim0287** [comment]
I've been using this Pi as a Pi-hole ad blocker and smbd server for a long time so it's a bit challenging to recall.  However I believe the only system changes I made were the locale to en_US and the timezone.

**Comment by Fourdee** [comment]
@slim0287 

> en_US and the timezone.

Thanks 👍 

If you get a chance, on the any of the SSH clients with this issue, once connected, please paste results:
```
echo $TERM
echo $LANG
echo $LC_ALL
```

**Comment by slim0287** [comment]
I connected with macOS.  The results were:
```
$ echo $TERM 
xterm-256color
$ echo $LANG
en_US.UTF-8
$ echo $LC_ALL
en_US.UTF-8
```

Any new insights?

Thanks

**Comment by Fourdee** [comment]
Commit sent, should be resolved with: https://github.com/Fourdee/DietPi/commit/6d20b08be0b43ee9ff4a17136ebdb4520e711352

Pending v6.9 release.

**Comment by Fourdee** [comment]
Patch test verified. I'll mark this as closed, however, @MichaIng if you want to reopen for other options, please do 👍

**Comment by Fourdee** [comment]
User error (me), reopened for testing.

**Comment by Fourdee** [comment]
@slim0287 

Would you be willing to help us test the following (copy and paste all):
```
rm /etc/profile.d/99-dietpi-login.sh
sed -i '/\/DietPi/d' /root/.bashrc #should already be removed, failsafe start clean
sed -i '/\/DietPi/d' /home/dietpi/.bashrc #should already be removed, failsafe start clean

cat << _EOF_ >> /root/.bashrc
. /DietPi/dietpi/func/dietpi-globals
/DietPi/dietpi/login
_EOF_
cat << _EOF_ >> /home/dietpi/.bashrc
. /DietPi/dietpi/func/dietpi-globals
/DietPi/dietpi/login
_EOF_

```

Once completed, reconnect via SSH and see if the error is removed, and, you see the DietPi login banner.

**Comment by slim0287** [comment]
@Fourdee That solved both issues-- the -sh: 146 Syntax error and the sudo su dietpi welcome information.  Thanks!

Slim

**Comment by Fourdee** [comment]
@slim0287 

Excellent, many thanks for testing and confirming fix. 👍 
This fix will be applied automatically during v6.9 update (ETA: 1+ week release).

I'll mark this as resolved.
